src/scripts/destinyTrackerApi/reviewDataCache.js

```diff
@@ -57,24 +86,52 @@ class reviewDataCache {
     matchingItem.rating = rating;
   }
 
+  /**
+   * Keep track of expanded item review data from the DTR API for this DIM store item.
+   * The expectation is that this will be building on top of community score data that's already been supplied.
+   *
+   * @param {any} item
+   * @param {any} reviewsData
+   *
+   * @memberof ReviewDataCache
+   */
   addReviewsData(item,
                  reviewsData) {
-    var matchingItem = this.getMatchingItem(item);
+    var matchingItem = this._getMatchingItem(item);
 
     matchingItem.reviews = reviewsData.reviews;
     matchingItem.reviewsDataFetched = true;
   }
 
+  /**
+   * Fetch the collection of review data that we've stored locally.
+   *
+   * @returns {array}
+   *
+   * @memberof ReviewDataCache
+   */
   getItemStores() {
     return this._itemStores;
   }
 
+  /**
+   * There's a 10 minute delay between posting an item review to the DTR API
+   * and being able to fetch that review from it.
+   * To prevent angry bug reports, we'll continue to hang on to local user review data for
+   * 10 minutes, then we'll purge it (so that it can be re-pulled).
+   *
+   * Item is just an item from DIM's stores.
+   *
+   * @param {any} item
+   *
+   * @memberof ReviewDataCache
+   */
   eventuallyPurgeCachedData(item) {
     var tenMinutes = 1000 * 60 * 10;
     var self = this;
 
     setTimeout(function() {
```

Comment:
Same comments about the task for purging the cache. This should be on the retrieval side, not a `setTimeout`.